control_protocol.md

```diff
@@ -330,17 +330,490 @@ A Component MUST also offer a list of all possibly callable methods in accordanc
+                                "items": {
+                                    "type": "string"
+                                }
+                            }
+                        ]
+                    },
+                    "required": true
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "object",
+                    "additionalProperties": true
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "set_parameters",
+            "params": [
+                {
+                    "name": "parameters",
+                    "schema": {
+                        "type": "object",
+                        "additionalProperties": true
+                    },
+                    "required": true
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "call_action",
+            "params": [
+                {
+                    "name": "action",
+                    "schema": {
+                        "type": "string"
+                    },
+                    "required": true
+                },
+                {
+                    "name": "_args",
+                    "schema": {
+                        "anyOf": [
+                            {
+                                "type": "array"
+                            },
+                            {
+                                "type": "array"
+                            },
+                            {
+                                "type": "null"
+                            }
+                        ]
+                    },
+                    "required": false
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {},
+                "required": true
+            }
+        },
+    ],
+    "components": {
+        "schemas": {}
+    }
+}
+:::
+
+
+
+#### Polling Actor
+
+An {ref}`control_protocol.md#Actor`, which supports regular polling of values, MUST implement these methods, additionally to those of an Actor.
+
+::: json
+{
+    "openrpc": "1.2.6",
+    "info": {
+        "title": "Polling Actor",
+        "version": "0.1.0"
+    },
+    "methods": [
+        {
+            "name": "start_polling",
+            "params": [
+                {
+                    "name": "polling_interval",
+                    "schema": {
+                        "anyOf": [
+                            {
+                                "type": "number"
+                            },
+                            {
+                                "type": "null"
+                            }
```

Comment:
What is the difference between supplying polling_interval=null and leaving out the argument entirely (required=false)?

Reply:
I think this is a consequence of the automatic generation by the python package based on the type hint `None | int`.

Reply:
I reworked them all manually.